gems/pending/VolumeManager/MiqLvm.rb

```diff
@@ -232,9 +237,11 @@ def getLvObj(vgObj, lvName, lv)
   end # def getLvObj
 
   def getSegObj(seg)
-    segObj = LvSegment.new(seg['start_extent'], seg['extent_count'], seg['type'], seg['stripe_count'])
-    segObj.metadata = seg['metadata'] if seg.key?('metadata')
-    segObj.pool     = seg['pool']     if seg.key?('pool')
+    device_id = (seg['device_id'] ? seg['device_id'].to_i : nil)
```

Comment:
Prefer `device_id = seg['device_id'].try(:to_i)`

Note: The `try` bits imply that ActiveSupport is available, which it *should* be, but might not be in tests... @roliveri If it's not directly available, are you ok with making it available?  I know in the past we avoided it because we bundled this code separately in that .zip-launcher-thing, but we don't do that anymore.